scheduler.go

```diff
@@ -209,6 +210,25 @@ func (s *Scheduler) removeByCondition(shouldRemove func(*Job) bool) {
 	}
 }
 
+// Remove First found Job by Tag
+func (s *Scheduler) RemoveJobByTag(tag string) {
+	jobindex := s.findJobsIndexByTag(tag)
+	// Remove Only if return valid index
+	if jobindex >= 0 {
+		s.jobs = removeAtIndex(s.jobs, jobindex)
+	}
+}
+
+// Find first job index by given string
+func (s *Scheduler) findJobsIndexByTag(tag string) int {
```

Comment:
This function could return `(int, error)` instead of just `int`
So, if the job is found: 
```go
return i, nil
```
else:
```go
return -1, ErrJobNotFound // Exported error
```

Reply:
ohh ok, it will made the code clearer, thank you for the suggestion.